# Ref Frame: base
# Target frame: right_upper_shoulder
u0: [0,-0,1]
p0: [0.0640272,-0.259027,0.129626]
# DQ: 
#  0.923879        -0        -0 -0.382684 0.0248029 0.0791396 -0.107404 0.0598794

# Ref Frame: base
# Target frame: right_lower_shoulder
u1: [0.707108,0.707105,4.89586e-12]
p1: [0.112818,-0.307818,0.399976]
# DQ: 
#  0.653281 -0.653281  0.270599 -0.270599  0.132615 0.0243818  -0.21593 0.0453667

# Ref Frame: base
# Target frame: right_upper_elbow
u2: [0.707105,-0.707108,4.89642e-12]
p2: [0.184942,-0.379943,0.399976]
# DQ: 
#   0.653281   0.270599   0.653281  -0.270599   0.153199 -0.0188327 -0.0449657   0.242464

# Ref Frame: base
# Target frame: right_lower_elbow
u3: [0.707108,0.707105,9.79283e-12]
p3: [0.370501,-0.565502,0.330976]
# DQ: 
#   0.653281  -0.653281   0.270599  -0.270599   0.242314   0.152752  -0.242698 -0.0264773

# Ref Frame: base
# Target frame: right_upper_forearm
u4: [0.707105,-0.707108,4.89686e-12]
p4: [0.44375,-0.638751,0.330976]
# DQ: 
#  0.653281  0.270599  0.653281 -0.270599  0.193384  0.123259 -0.103822   0.33948

# Ref Frame: base
# Target frame: right_lower_forearm
u5: [0.707108,0.707105,1.46894e-11]
p5: [0.635163,-0.830166,0.320976]
# DQ: 
#   0.653281  -0.653281   0.270599  -0.270599   0.363219   0.276363  -0.290072 -0.0803848

# Ref Frame: base
# Target frame: right_wrist
u6: [0.707105,-0.707108,4.89686e-12]
p6: [0.71717,-0.912173,0.320976]
# DQ: 
#  0.653281  0.270599  0.653281 -0.270599  0.244348  0.252829 -0.157492  0.462517

pe_init_right: [0.653281,  0.270599,  0.653281, -0.270599,  0.259711,  0.289919, -0.172855,  0.499607] #THIS IS FOR right_hand as pe_init_right
# pe_init_right: [0.2706,   -0.2706,    0.6533,   -0.6533,    0.5600,    0.3604,    0.0965,    0.1792] #THIS IS FOR ar10_right/palm_centre as pe_init_right
# Ref Frame: base
# Target frame: right_hand
# u_right_7: [0.707105,-0.707108,4.89686e-12]
# p: [0.797462,-0.992465,0.320976]
# DQ: 
#  0.653281  0.270599  0.653281 -0.270599  0.259711  0.289919 -0.172855  0.499607

# Ref Frame: base
# Target frame: left_upper_shoulder
u7: [0,0,1]
p7: [0.0640272,0.259027,0.129626]
# DQ: 
#   0.923879          0          0   0.382684 -0.0248029  0.0791396   0.107404  0.0598794

# Ref Frame: base
# Target frame: left_lower_shoulder
u8: [-0.707108,0.707105,4.89586e-12]
p8: [0.112818,0.307818,0.399976]
# DQ: 
#   0.653281  -0.653281  -0.270599   0.270599  0.0243818   0.132615 -0.0453667    0.21593

# Ref Frame: base
# Target frame: left_upper_elbow
u9: [0.707105,0.707108,4.89653e-12]
p9: [0.184942,0.379943,0.399976]
# DQ: 
#   0.653281  -0.270599   0.653281   0.270599  -0.153199 -0.0188327  0.0449657   0.242464

# Ref Frame: base
# Target frame: left_lower_elbow
u10: [-0.707108,0.707105,9.79339e-12]
p10: [0.370501,0.565502,0.330976]
# DQ: 
#  0.653281 -0.653281 -0.270599  0.270599  0.152752  0.242314 0.0264773  0.242698

# Ref Frame: base
# Target frame: left_upper_forearm
u11: [0.707105,0.707108,4.89631e-12]
p11: [0.44375,0.638751,0.330976]
# DQ: 
#  0.653281 -0.270599  0.653281  0.270599 -0.193384  0.123259  0.103822   0.33948

# Ref Frame: base
# Target frame: left_lower_forearm
u12: [-0.707108,0.707105,1.46896e-11]
p12: [0.635163,0.830166,0.320976]
# DQ: 
#  0.653281 -0.653281 -0.270599  0.270599  0.276363  0.363219 0.0803848  0.290072

# Ref Frame: base
# Target frame: left_wrist
u13: [0.707105,0.707108,4.89631e-12]
p13: [0.71717,0.912173,0.320976]
# DQ: 
#  0.653281 -0.270599  0.653281  0.270599 -0.244348  0.252829  0.157492  0.462517
pe_init_left: [0.653281, -0.270599,  0.653281,  0.270599, -0.259711,  0.289919,  0.172855,  0.499607] #THIS IS FOR left_hand as pe_init_left
# pe_init_left: [0.6533,   -0.6533,    0.2706,   -0.2706,    0.1792,    0.0965,    0.3604,    0.5600]  #THIS IS FOR ar10_left/palm_centre as pe_init_left
# Ref Frame: base
# Target frame: left_hand
# u_left_7: [0.707105,0.707108,4.89631e-12]
# p: [0.797462,0.992465,0.320976]
# DQ: 
#  0.653281 -0.270599  0.653281  0.270599 -0.259711  0.289919  0.172855  0.499607

joint_type: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
joint_names: ['right_s0', 'right_s1', 'right_e0', 'right_e1', 'right_w0', 'right_w1', 'right_w2', 'left_s0', 'left_s1', 'left_e0', 'left_e1', 'left_w0', 'left_w1', 'left_w2']

root_name: 'torso'
tip_name_left: 'left_hand'
tip_name_right: 'right_hand'